ui/src/containers/AppsTableContainer.js

```diff
@@ -15,22 +16,22 @@ export class AppsTableContainer extends React.Component {
   componentWillMount () {
     this.props.getApps();
   }
-  onRowClick (event, rowId, props) {
+  onRowClick (_event, rowId) {
     var app = this.props.apps.data.filter(app => {
       return app.appName === rowId[0];
     });
     const id = app[0].id;
     const path = '/app/' + id;
     this.props.history.push(path);
   }
-  onSort (_event, index) {
-    this.props.reverseAppsTableSort(index);
+  onSort (_event, index, direction) {
```

Comment:
Is `_event` param required (as it's not being used)

Reply:
It is the first parameter in a default function, it cannot be removed.